packages/qdrant-loader/src/qdrant_loader/connectors/publicdocs/connector.py

```
@@ -306,16 +314,36 @@ async def get_documents(self) -> list[Document]:
                     self.logger.error(f"Failed to process page {page}: {e}")
                     continue
 
-            if not documents:
+            if yielded_count == 0:
                 self.logger.warning("No valid documents found to process")
-                return []
-
-            return documents
 
         except Exception as e:
             self.logger.error("Failed to get documentation", error=str(e))
```

Ruff (0.15.15)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)